package.json

```diff
@@ -93,7 +93,7 @@
     "@storybook/react": "^8.0.10",
     "@storybook/react-vite": "^8.0.10",
     "@storybook/test": "^8.0.10",
-    "@terra-ui-packages/test-utils": "*",
+    "@terra-ui-packages/test-utils": "1.0.0",
```

Comment:
For what it's worth, private packages (such as the Terra UI application) _can_ use `*`.